bentoml/_internal/frameworks/catboost.py

```diff
@@ -0,0 +1,276 @@
+from __future__ import annotations
+
+import os
+import typing as t
+import logging
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+import bentoml
+from bentoml import Tag
+from bentoml.exceptions import NotFound
+from bentoml.exceptions import InvalidArgument
+from bentoml.exceptions import MissingDependencyException
+from bentoml._internal.models.model import ModelContext
+
+from ..utils.pkg import get_pkg_version
+
+if TYPE_CHECKING:
+    from bentoml.types import ModelSignature
+    from bentoml.types import ModelSignatureDict
+
+    from .. import external_typing as ext
+
+try:
+    import catboost as cb
+except ImportError:  # pragma: no cover
+    raise MissingDependencyException(
+        """catboost is required in order to use module `bentoml.catboost`, install
+        catboost with `pip install catboost`. For more information, refers to
+        https://catboost.ai/en/docs/concepts/installation
+        """
```

Comment:
```suggestion
        '"catboost" is required in order to use module `bentoml.catboost`, install catboost with `pip install catboost`. For more information, refers to https://catboost.ai/en/docs/concepts/installation'
```

Reply:
@aarnphm Copied from xgboost.py

Reply:
I guess we want to write docs on single line, but it renders as a single line nonetheless.